tests/PHPStan/Type/UnionTypeTest.php

```diff
@@ -1186,6 +1188,72 @@ public function dataAccepts(): array
 			],
 
 		];
+
+		if (PHP_VERSION_ID < 80100) {
+			return;
```

Comment:
It's a bit tricky to mark the rest of the dataProvider as tested only on PHP 8.1. Please reverse the condition and put the enum `yield`s into that condition:

```php
if (PHP_VERSION_ID >= 80100) {
    yield...
}
```

If PHPCS complains, move one existing yield below the condition.

Reply:
I reversed the condition and moved some of the existing yields.